docs/lsm/partial_compiles/partial_compiles.rst

```diff
@@ -26,11 +26,13 @@ Supported scenarios
 Partial compiles are possible when
 
 1. Service Instances are unrelated: service instances don't share any resources and don't depend on each other in any way. This only requires correctly setting :inmanta:relation:`owned_resources<lsm::ServiceBase.owned_resources>`.
-2. Services form groups under a common owner. 
-    Instances within the group can freely depend on each other and share resources, but nothing is shared across groups. 
-    One specific instance is designated as the common owner of the group.
-    This additionally requires indicating what the owner of any service is, by setting :inmanta:relation:`lsm::ServiceEntityBinding.owner` and :inmanta:attribute:`lsm::ServiceEntityBinding.relation_to_owner`.
-    This does not immediately have to be the root owner, the ownership hierarchy is allowed to form a tree with intermediate owners below the root owner. 
+2. Services form groups under a common owner.
+
+   - Instances within the group can freely depend on each other and share resources, but nothing is shared across groups.
+   - One specific instance is designated as the common owner of the group.
+   - Instances can not be moved to another group. The model should prevent this type of update.
```

Comment:
When building services, making updates secure is always something you have to do, so it is just one more constraint. 

I think it is not the time to put too many constraints in lsm, because we don't know what the relevant usecases are. 
It would be difficult to check and possibly over conservative.